REFACTORING.md

```diff
@@ -0,0 +1,175 @@
+# Refactoring Documentation: Removal of profi.workloads
+
+## Summary
```

Comment:
a dedicated documentation for this change is not necessary. i have already removed this file